Fixes laptop and tweaks stacks

LetterN opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Fixes laptop pickup() and stacks can be split while on floor. Also (possibly) fixes DIy shuttle

closes https://github.com/Citadel-Station-13/Citadel-Station-13/issues/14429
closes https://github.com/Citadel-Station-13/Citadel-Station-13/issues/14424
fixes https://github.com/Citadel-Station-13/Citadel-Station-13/issues/14452
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
tweak: disables the istile() check on stacks
fix: fixed laptops pickability
fix: fixed closets being unweldable
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

LetterN committed:
laptop and closet decon fix

SandPoot commented on code/game/objects/structures/crates_lockers/closets.dm:
```diff
@@ -354,61 +272,74 @@
 	if(opened)
-		if(istype(W, cutting_tool))
-			var/welder = FALSE
+		if(istype(W, /obj/item/weldingtool))
```
See if you can take this off so the tool behavior is used.

LetterN committed:
fixes https://github.com/Citadel-Station-13/Citadel-Station-13/issues/14452
and-might-have fi-x-ed https://github.com/Citadel-Station-13/Citadel-Station-13/issues/14450

LetterN committed:
Merge remote-tracking branch 'origin/master' into fixes-2

LetterN committed:
cbt

LetterN committed:
Merge remote-tracking branch 'origin/master' into fixes-2
